src/theme.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  _name: 'network-dashboard',
+  _appearance: 'light',
+}
```

Comment:
This could be an easy way to use the new primary brand colours without having to update aragon/ui immediately.